examples/swap/swap-e2e.html

```diff
@@ -95,12 +96,12 @@ <h3>Claim swap:</h3>
     /* global $, ChainAbstractionLayer, web3 */
     const { Client, providers, networks, crypto } = ChainAbstractionLayer
     const bitcoin = new Client()
-    bitcoin.addProvider(new providers.bitcoin.BitcoinRPCProvider('https://localhost:18332', 'bitcoin', 'local321'))
-    bitcoin.addProvider(new providers.bitcoin.BitcoinLedgerProvider({ network: networks.bitcoin_testnet }))
+    bitcoin.addProvider(new providers.bitcoin.BitcoinRPCProvider('http://localhost:8000', 'bitcoin', 'local321'))
+    bitcoin.addProvider(new providers.bitcoin.BitcoinLedgerProvider({ network: networks.bitcoin_testnet, segwit: false }))
```

Comment:
if you made `segwit` default to false then won't need to specify it

Reply:
ah yes good point there

Reply:
ah yes good point. Although you could also turn this into regular arguments instead of an object and it would work that way
Not sure if you had some intentions with using an object here

Reply:
ah yes good point. Although you could also turn this into regular arguments instead of an object and it would work that way
Not sure if you had some intentions with using an object here

Reply:
ah yes good point. Although you could also turn this into regular arguments instead of an object and it would work that way
Not sure if you had some intentions with using an object here